cows.txt

```diff
@@ -5098,3 +5098,49 @@ mmm   | |  |              \ \ |  |
 *  ||----||
 Cow Police Officer
 (Without gun and handcuffs)
+
+
+                                                           o  o
+                                                           \__/
+    _______________________________________________________(oo)
+  /|                                                       |\/
+ / |---|---|---|---|---|---|---|---|---|---|---|---|---|---||
+*  ||  ||  ||  ||  ||  ||  ||  ||  ||  ||  ||  ||  ||  ||  ||  
+Cow Centipede
+
+
+          (__)     (__)     (__)
+  /-------(o|------(o|------(oo)
+ / |     ||\|     ||\|     ||\/
+*  ||----||*||----||*||----||
+   ~~    ~~ ~~    ~~ ~~    ~~
+Cow Centipede
```

Comment:
I don't really see the point of this. It's already covered more correctly above.